content/md/first-steps/manage-your-rules.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: manage-your-rules
 themes: boost-your-productivity, catalog-settings
-title: Get started with the **rules engine**
+title: Manage your **rules**
 popular: false
 ee-only: true
 related: what-is-a-rule
 ---
 
 # Overview
 
-A rule allows you to enrich automatically some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
-For more details, please refer to [What is a rule](what-is-a-rule.html).
+A rule allows you to automatically enrich some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  
+For more details, please refer to [the dedicated article](blabla.html).
```

Comment:
Does the page "blabla.html" exists?

Reply:
Nope, I'll add the link afterward. This is the second article that should be soon ready ;)